native_client/ctcdecode/Makefile

```diff
@@ -17,12 +17,13 @@ endif
 
 all: bindings
 
-clean-keep-common:
+clean-keep-archive:
```

Comment:
Name this `clean-keep-third-party`, and it needs to delete the first party archive.